Tools/autotest/arducopter.py

```diff
@@ -3848,39 +3840,31 @@ def RangeFinder(self):
         m = self.assert_receive_message('RANGEFINDER', timeout=10)
 
         self.progress("Checking RangeFinder is marked as enabled in mavlink")
-        m = self.mav.recv_match(type='SYS_STATUS',
-                                blocking=True,
-                                timeout=10)
+        m = self.assert_receive_message('SYS_STATUS', timeout=10)
         flags = m.onboard_control_sensors_enabled
         if not flags & mavutil.mavlink.MAV_SYS_STATUS_SENSOR_LASER_POSITION:
             raise NotAchievedException("Laser not enabled in SYS_STATUS")
         self.progress("Disabling laser using switch")
         self.set_rc(9, 1000)
         self.delay_sim_time(1)
         self.progress("Checking RangeFinder is marked as disabled in mavlink")
-        m = self.mav.recv_match(type='SYS_STATUS',
-                                blocking=True,
-                                timeout=10)
+        m = self.assert_receive_message('SYS_STATUS', timeout=10)
         flags = m.onboard_control_sensors_enabled
         if flags & mavutil.mavlink.MAV_SYS_STATUS_SENSOR_LASER_POSITION:
             raise NotAchievedException("Laser enabled in SYS_STATUS")
 
         self.progress("Re-enabling rangefinder")
         self.set_rc(9, 2000)
         self.delay_sim_time(1)
-        m = self.mav.recv_match(type='SYS_STATUS',
-                                blocking=True,
-                                timeout=10)
+        m = self.assert_receive_message('SYS_STATUS', timeout=10)
         flags = m.onboard_control_sensors_enabled
         if not flags & mavutil.mavlink.MAV_SYS_STATUS_SENSOR_LASER_POSITION:
             raise NotAchievedException("Laser not enabled in SYS_STATUS")
 
         self.takeoff(10, mode="LOITER")
 
-        m_r = self.mav.recv_match(type='RANGEFINDER',
-                                  blocking=True)
-        m_p = self.mav.recv_match(type='GLOBAL_POSITION_INT',
-                                  blocking=True)
+        m_r = self.assert_receive_message('RANGEFINDER')
```

Comment:
I'm not sure how much it matters but the default timeout on recv_match was None and by moving to assert_receive_message the default is 1.  That is probably for the best but may lead to a few new flapping tests if 1 s is a bit of a tight tolerance.  Just an observation though.

Reply:
> I'm not sure how much it matters but the default timeout on recv_match was None and by moving to assert_receive_message the default is 1. That is probably for the best but may lead to a few new flapping tests if 1 s is a bit of a tight tolerance. Just an observation though.

My reading was that if it comes in as `None` then it will get defaulted to `1` explicitly further down in teh call train.  One of us is wrong :-)